ansible/environments/mac/group_vars/all

```diff
@@ -1,4 +1,4 @@
-whisk_version_name: local
+whisk_version_name: mac
 nginx_conf_dir: /tmp/nginx
```

Comment:
/Users/Shared should be used instead for these because it allows the logs etc to be accessible from the mac host.